modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,52 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+#TODO: Clean this up to call the scripts instead
+
+shopt -s globstar
+echo "Setting environment variables"
+jarfile="/deployments/*.jar"
+libdir="/deployments/lib"
+$JAVA_HOME/
+
+echo "Checking that jarfile and libdir exist"
+test -f "$jarfile"
+test -d "$libdir"
```

Comment:
the script dies at this point for me with a test quarkus "fat-jar" app, because there is no libdir.